int32 vid
int32 src
int32 dst
---
int32 next_hop_ip